modules/metadata/pages/discover.adoc

```diff
@@ -18,7 +18,7 @@ Example:
 +
 [source]
 --
-IMAGE=quay.io/redhat-user-workloads/workspace-tenant/application/component@sha256:<output omitted>
+IMAGE=quay.io/redhat-user-workloads/namespace-tenant/application/component@sha256:<output omitted>
```

Comment:
The image pullspecs you have are not quite correct. I think that this is just the namespace. All of our namespaces just have the `-tenant` suffix.